media-gfx/openscad/openscad-2019.05.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit elisp-common qmake-utils xdg
+
+# 2019.05
+#MY_VER=$(ver_cut 1-2) # version component
+#MY_REL=$(ver_cut 4) # release component, 'p' being the third component
+#MY_PV=${MY_VER}-${MY_REL}
+SITEFILE="50${PN}-gentoo.el"
+
+DESCRIPTION="The Programmers Solid 3D CAD Modeller"
+HOMEPAGE="http://www.openscad.org/"
+SRC_URI="https://github.com/${PN}/${PN}/releases/download/${P}/${P}.src.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="emacs"
+
+# FIXME: add optional lib3mf hidapi spnav
+RDEPEND="
+	dev-cpp/eigen:=
+	dev-libs/boost:=
+	dev-libs/double-conversion:=
+	dev-libs/glib:2=
+	dev-libs/gmp:0=
+	dev-libs/libzip:=
+	dev-libs/mpfr:0=
+	dev-qt/qtconcurrent:=
+	dev-qt/qtcore:=
+	dev-qt/qtdbus:=
+	dev-qt/qtgui:=[-gles2]
+	dev-qt/qtmultimedia:=
+	dev-qt/qtnetwork:=
+	dev-qt/qtopengl:=
+	dev-qt/qtwidgets:=
+	media-gfx/opencsg
+	media-libs/fontconfig:=
+	media-libs/freetype:=
+	>=media-libs/glew-2.0.0:*
+	media-libs/harfbuzz:=
+	sci-mathematics/cgal:=
+	x11-libs/qscintilla:=
```

Comment:
This is not wrong when you look at the tree today, but it is suboptimal for upgraders coming from a rough year ago. You've got two options here:
- Drop the minimum version (as you did) but keep `[qt5(+)]`
- Drop `[qt5(+)]` but raise minimum version to `>=2.10.3` (this was the version that dropped qt5 flag)

Reply:
I'd take the latter approach but use `>=2.10.8`, the lowest stable version currently in the tree. Would this be ok?

Reply:
Better to document the real minimum version from portage's POV, when you look at it again in, say, one year's time, you'll know exactly what version to lookup in git (to determine if it is ok to drop the min version).

Reply:
Good point! Will it be enough to document this in a comment?

Reply:
Thinking about it... documenting this in a comment could be rather confusing. I'm using the 2.10.3 version then.